README.md

```diff
@@ -193,7 +193,6 @@ matters to you.
 
 ## Support
 - [Fastify help](https://github.com/fastify/help)
```

Comment:
Can you add some text before this link? A list of only 1 element is not nice.

Reply:
Updated.